apps/internal/base/internal/storage/items_test.go

```diff
@@ -54,14 +55,16 @@ func TestCreateAccessToken(t *testing.T) {
 		testExtExpiresOn,
 		"user.read",
 		"access",
+		"tokenType",
+		"keyID",
 	)
 	if !extExpiresOn.Equal(actualAt.ExtendedExpiresOn.T) {
 		t.Errorf("Actual ext expires on %s differs from expected ext expires on %s", actualAt.ExtendedExpiresOn, extExpiresOn)
 	}
 }
 
 func TestKeyForAccessToken(t *testing.T) {
-	const want = "testhid-env-accesstoken-clientid-realm-user.read"
+	const want = "testhid-env-accesstoken-clientid-realm-user.read-bearer"
```

Comment:
Can we make sure (via a test) that the old format is usable? I.e. if I have a token cache item without "-bearer" in the key, MSAL can read it and know that this is a bearer token.